LibOS/shim/test/native/Makefile

```diff
@@ -27,12 +27,7 @@ static: %: %.c
 	@$(CC) $(CFLAGS) -o $@ -static $< \
 	$(shell echo $@ | sed 's/^[^\.]*//g' | sed 's/\./ -l/g')
 
-shared_object: %: %.c
-	@echo [ $@ ]
-	@$(CC) $(CFLAGS) -o $@ -fPIC -pie $< \
-	$(shell echo $@ | sed 's/^[^\.]*//g' | sed 's/\./ -l/g')
-
-pie: %: %.c
+xopie: %: %.c
```

Comment:
Is there any reason of disabling the rule for pie?

Reply:
No, this looks like a typo.  I only intended to take out the shared_object rule.